### ISSUE TITLE
Start screen on resume prevents login to accounts

### ISSUE THREAD

**Issue by tma** [cryptomator/cryptomator-ios#62]
First of all: Great piece of software 👍. Thank you for creating and providing it! ❤️

### Basic Info

- I'm running Cryptomator on: iPhone
- I'm using Cryptomator in version: 1.2.0

### Description

Hi 👋. I have a vault on Google Drive and on Dropbox. For both accounts there I have a random password stored in 1Password plus I have 2-factor authentication enabled. That means, to login I'd have to jump to 1Password and back to Cryptomator, and once again to my 2FA app and back to Cryptomator. That's all fine since it would be only necessary once.

The problem is though that the Cryptomator app always jumps back to the start/home screen (list of vaults) after the app is resumed. This makes it impossible for me to login on my phone when it's also my 2FA device 😞.

I wonder if you could change something when the Cryptomator app gets resumed and not jump back to the start screen? It would be an improvement in general.

**Comment by tobihagemann** [comment]
Hmmm... you might be on to something. What is your "Auto-Lock" set to in the settings of Cryptomator (gear icon at the top right)? If it's set to "Instant", temporarily set it to e.g. "1 Minute".

**Comment by tma** [comment]
Yep, was "Instant". A temporary switch did prevent the jump to the home screen and let me sign in 🎉. Thanks for your help! 🙇 

Closing this issue. Feel free to reopen in case you want to use it for tracking an improvement for that case.

**Comment by tobihagemann** [comment]
Okay, great! I guess, I'll reopen this issue nevertheless, because this behavior is not intended. 😆 It'll have a lower priority because my impression is that it occurs rarely, due to "1 Minute" being default and logging in (after setting auto-lock to "Instant") not being a frequent operation.

I'm just not so sure if it can be easily fixed, because the current implementation is rather crude. 😅

Thank you for reporting this issue. :grin::+1:
